app/src/forms/form/service.js

```diff
@@ -281,26 +281,6 @@ const service = {
     }
   },
 
-  updateSubmission: async (formSubmissionId, data, currentUser) => {
```

Comment:
I'm not sure if we should be removing this function yet? We'll likely be using this in the future when you end up editing a submission (submitter and reviewer back and forth interaction). It'd just be missing the logic to append to a log table (doesn't exist yet obviously).

Reply:
Yeah it's unused in the form service but definitely will be used in the submission service (duplicated there so removed this one)